### PR TITLE
fix 318

### PR DESCRIPTION
**Pull request by arifnyet**

### CODE REVIEW

File: src/main/java/com/livingprogress/mentorme/controllers/MentorController.java [hunk 1]
```diff
@@ -231,7 +232,7 @@ public void delete(@PathVariable long id) throws MentorMeException {
      */
     @RequestMapping(method = RequestMethod.GET)
     public SearchResult<Mentor> search(@ModelAttribute MentorSearchCriteria criteria,
-            @ModelAttribute Paging paging) throws MentorMeException {
+                                       @ModelAttribute Paging paging) throws MentorMeException {
```
Comment (huangqun) [inline src/main/java/com/livingprogress/mentorme/controllers/MentorController.java]: Please keep the original code indention here.

File: src/main/java/com/livingprogress/mentorme/controllers/MentorController.java [hunk 2]
```diff
@@ -261,7 +262,7 @@ public int getAverageScore(@PathVariable long id) throws MentorMeException {
      */
     @RequestMapping(value = "{id}/matchingMentees", method = RequestMethod.GET)
     public List<Mentee> getMatchingMentees(@PathVariable long id,
-            @ModelAttribute MatchSearchCriteria matchSearchCriteria) throws MentorMeException {
+                                           @ModelAttribute MatchSearchCriteria matchSearchCriteria) throws MentorMeException {
```
Comment (huangqun) [inline src/main/java/com/livingprogress/mentorme/controllers/MentorController.java]: Please keep the original code indention here.

File: src/main/java/com/livingprogress/mentorme/controllers/MentorController.java [hunk 3]
```diff
@@ -323,7 +327,7 @@ public int getAverageScore(@PathVariable long id) throws MentorMeException {
      */
     @RequestMapping(value = "{id}/remoteMatchingMentees")
     public List<Mentee> remoteMatchingMentees(@PathVariable long id,
-            @ModelAttribute MatchSearchCriteria criteria) throws MentorMeException {
+                                              @ModelAttribute MatchSearchCriteria criteria) throws MentorMeException {
```
Comment (huangqun) [inline src/main/java/com/livingprogress/mentorme/controllers/MentorController.java]: Please keep the original code indention here

File: src/main/java/com/livingprogress/mentorme/utils/Helper.java [hunk 4]
```diff
@@ -538,6 +539,35 @@ public static Predicate buildNamePredicate(String name, Predicate pd, Root<?> ro
         return resultPD;
     }
 
+    public  static double distance(double lat1, double lon1, double lat2, double lon2) {
```
Comment (huangqun) [inline src/main/java/com/livingprogress/mentorme/utils/Helper.java]: Method name should be not be a noun, make it calculateDistance or something.

**Comment by huangqun** [comment]
Also fix the code formatting and documentation for these and re-submit the PR:

image

Thanks!

**Comment by arifnyet** [comment]
@huangqun it is calculated from latitude and longitude value retrieved from the database as there is a service updating them... "We have a batch process that runs nightly and geocodes new users based on the addresses that they've provided." so i assume the lat and lon of each user is updated daily...

**Comment by huangqun** [comment]
@arifnyet ok, please fix the issues and submit a new PR then.

**Pull request closed without merging**